package.json

```diff
@@ -23,8 +23,8 @@
     "accessibility-tests-drone": "node node_modules/axe-cli/axe-cli --exit --disable page-has-heading-one,meta-viewport",
     "install": "cd node_modules/cucumber; yarn install --frozen-lockfile",
     "acceptance-tests": "cucumber-js --require-module @babel/register --require-module @babel/polyfill --require tests/acceptance/setup.js --require tests/acceptance/stepDefinitions --format node_modules/cucumber-pretty -t \"${TEST_TAGS:-not @skip and not @skipOnOC10}\"",
-    "acceptance-tests-ocis": "NODE_TLS_REJECT_UNAUTHORIZED=0 RUN_ON_OCIS=true cucumber-js --require-module @babel/register --require-module @babel/polyfill --require tests/acceptance/setup.js --require tests/acceptance/stepDefinitions --format node_modules/cucumber-pretty -t \"${TEST_TAGS:-not @skip and not @skipOnOCIS and not @notToImplementOnOCIS}\"",
-    "acceptance-tests-with-ldap": "RUN_ON_OCIS=true RUN_WITH_LDAP=true cucumber-js --require-module @babel/register --require-module @babel/polyfill --require tests/acceptance/setup.js --require tests/acceptance/stepDefinitions --format node_modules/cucumber-pretty -t \"${TEST_TAGS:-not @skip and not @skipOnOCIS and not @notToImplementOnOCIS}\"",
+    "acceptance-tests-ocis": "WEB_UI_CONFIG='./tests/acceptance/ocis-mac-config.json' SERVER_HOST='https://host.docker.internal:9200' BACKEND_HOST='https://host.docker.internal:9200' NODE_TLS_REJECT_UNAUTHORIZED=0 RUN_ON_OCIS=true cucumber-js --require-module @babel/register --require-module @babel/polyfill --require tests/acceptance/setup.js --require tests/acceptance/stepDefinitions --format node_modules/cucumber-pretty -t \"${TEST_TAGS:-not @skip and not @skipOnOCIS}\"",
```

Comment:
I don't see any `tests/acceptance/ocis-mac-config.json` in this repo, and it is not added in this PR.
I guess this is for running locally on a Mac? 
What is the intention here? Or is this accidentally committed?

Reply:
it is there. I didnt add it 🤷

Reply:
I must be going crazy. It is there! I was probably looking in some other `tests/acceptance` directory in some other app.